paparazzi/build.gradle

```diff
@@ -45,7 +45,7 @@ dependencies {
   api libs.junit
   api libs.androidx.annotations
   api libs.guava
-  api libs.kotlinx.coroutines.core
+  api libs.kotlinx.coroutines.android
```

Comment:
it appears that there used to be a transitive dependency on coroutines.android and this bump caused this reference to [Handler.asCoroutineDispatcher](https://github.com/cashapp/paparazzi/blob/master/paparazzi/src/main/java/app/cash/paparazzi/PaparazziSdk.kt#L588) to fail.